src/components/userProfile/components/SideMenu.js

```diff
@@ -17,9 +17,9 @@ export default class SideMenu extends React.Component {
                 mode={INLINE}
             >
                 {
-                    SIDE_MENU_ITEMS.map(sideMenuItem => (
-                        <Menu.Item key={sideMenuItem.label}>
-                            <span><Icon type={sideMenuItem.iconType} />{sideMenuItem.label}</span>
+                    SIDE_MENU_ITEMS.map(({ path, iconType, label }) => (
```

Comment:
the objects in `SIDE_MENU_ITEMS` do not contain a `path` key.  remove it from this piece of logic and use `label` as the `key` for the `Menu.Item`s

Reply:
OK